pypeit/spectrographs/keck_deimos.py

```diff
@@ -716,6 +718,28 @@ def bpm(self, filename, det, shape=None, msbias=None):
 
         return bpm_img
 
+    def get_lamps(self, fitstbl):
```

Comment:
Is there anything in this function that's specifically tied to DEIMOS?  I.e., apart from enforcing that this approach only be supported for DEIMOS, is there a reason we can't make this an attribute of the `Spectrograph` base class?

Reply:
Actually, It is somewhat DEIMOS specific, because it depends on how the lamps are called in the header of the specific instrument and how is translated in "PypeIt language". 
As you can see line 739 is doing this conversion, i.e., taking the name of the lamp from the header and adding a "I", which works for all the DEIMOS lamps, but it's a bit more complicated for some of the other instruments.

Reply:
Thanks for checking!  That makes perfect sense.